ui/webui/src/components/Error.jsx

```diff
@@ -53,6 +53,21 @@ export const bugzillaPrefiledReportURL = (productQueryData) => {
     return reportURL.href;
 };
 
+const ensureMaximumReportURLLength = (reportURL) => {
+    const newUrl = new URL(reportURL);
+    const searchParamsLimits = [
+        { param: "short_desc", length: 256, message: null },
+        { param: "comment", length: 7800, message: "\nTHE COMMENT EXCEEDING THE LENGTH LIMIT WAS TRUNCATED" },
```

Comment:
Should not there be an error in the dialog form instead of filling this warning in the URL?

Reply:
The comment (pre-filed Detail) is generated by Anaconda, there is no user input at this point (currently it is used only in CriticalError where it is the exception content) so I don't think it makes sense to inform user about it. It is meant as an information for developers, not for the user. So maybe we should remove it completely (and just log it) so it does not confuse user in the BZ report form.

Reply:
I just realized, there is no mention of what's the actual limit? What is 7800?

Reply:
> I just realized, there is no mention of what's the actual limit? What is 7800?

I'll add a comment in the code.